docs/_includes/attr-charref.adoc

```diff
@@ -125,7 +125,11 @@ The character references (e.g., \&#34;) are used whenever the Unicode character
 The idea is to output text that just works.
 It's up to the converter to transform this text into something the viewer understands (for example, in the case of manpage or PDF output).
 
-^[2]^ The Zero Width Space (ZWSP) is a code point in Unicode that shows where a long word can be split if necessary.
+^[2]^ Asciidoctor does not prevent you from reassigning these attributes, but it is better to treat them as read-only.
 
-^[3]^ The word joiner (WJ) is a code point in Unicode prevents a line break at its position.
+^[3]^ You can also use any of the 252 HTML 4 character entity references, for example \&euro; gives &euro;
```

Comment:
This statement is not entirely true. It highly depends on the output format. What is true is that Asciidoctor passes named entities through. If you're generating HTML, then any of the named HTML 4 entities will be recognized by the viewer (i.e., browser). However, this causes problems when generating non-HTML output because it relies on the entities being defined in that toolchain. It particularly break down when creating PDF.

In general, our recommendation is to avoid using named entities except for the ones defined in XML (lt, gt, amp, quot, apos) (hence why we define them here).

Therefore, I'd change the footnote to say the following:

> Asciidoctor will pass through any named entity to the output document (e.g., \&euro;). However, it is up to the interpreter (e.g., browser) to recognize these entities. Aside from the named entities defined in XML (lt, gt, amp, quot, apos), we recommend always using the numeric character reference equivalent for the best universal support.